openrouteservice/src/main/java/org/heigit/ors/fastisochrones/partitioning/FastIsochroneParameters.java

```diff
@@ -0,0 +1,74 @@
+package org.heigit.ors.fastisochrones.partitioning;
+
+/**
+ * Parameters for fast isochrone algorithm preprocessing and query processing. Some defaults can be changed via app.config
+ *
+ * @author Hendrik Leuschner
+ */
+public final class FastIsochroneParameters {
+    //NAMES
+    public static final String ACTIVECELLDIJKSTRA = "ActiveCellDijkstra";
+    public static final String CORERANGEDIJKSTRA = "CoreRangeDijkstra";
+    public static final String RANGEDIJKSTRA = "RangeDijkstra";
+
+    //GLOBAL
+    private static int maxThreadCount = 12;
+    private static boolean log = true;
+    //PARTITIONING
+    private static int maxCellNodesNumber = 5000;
+    private static int minCellNodesNumber = 4;
+    private static double splitValue = 0.2525;
```

Comment:
Parameter was set by Stefan in his Masters thesis. He did some tests on it.

Reply:
OK, yet I find it weird to have this specific value in code without any background info why it has been chosen like this, don't you think?

Reply:
Yes, the same applies to each hard coded value in the fast isochrones processing. For each value, I have done some evaluations on what is good. They are not parameter sweeped on a miniscule scale to find optima, because there is interaction between parameters, too. I am aware that its nice to have all this, but if I want to give a working algorithm to the customer, I cannot spend months creating graphs and analyzing what optimal value each parameter has. I can put in a comment referencing the master's thesis here, but it will not solve the issue for other values (maxCellNodesNumber, MAX_SUBCELL_NUMBER, CONSIDERED_PROJECTIONS, minCellNodesNumber, the number and choice of projections, etc.).

Reply:
I will try and add the sources for parameters where they are available to me.